spec/views/spree/layouts/admin/_login_nav.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'spree/layouts/admin/_login_nav.html.haml' do
+
+  it 'display the application version in admin navbar' do
+    def  view.spree_current_user
+      FactoryGirl.build(:user)
+    end
```

Comment:
I understood that `assign` can give a value to instance variables and inject them into the view template during the test, as if they were coming from the controller.

But here we tried to emulate a helper: `spree_current_user`
I used what I could understand [in this discussion](https://github.com/rspec/rspec-rails/issues/1076).